Makefile

```diff
@@ -227,3 +227,19 @@ $(patsubst %,build/%.go,$(BENCHMARKS)): build/%.go: %.py $(COMPILER)
 $(BENCHMARK_BINS): build/benchmarks/%_benchmark: build/benchmarks/%.go $(RUNTIME) $(STDLIB)
 	@mkdir -p $(@D)
 	@go build -o $@ $<
+
+# ------------------------------------------------------------------------------
+# Installation
+# ------------------------------------------------------------------------------
+
+install:
```

Comment:
This rule should depend on $(RUNNER_BIN) $(COMPILER) $(RUNTIME) $(STDLIB) so that all the necessary files are present in the build subdir.